common.mk

```diff
@@ -114,3 +116,6 @@ generate-manifests:
 
 fmt: $(GOIMPORTS) ## Run goimports
 	$(GOIMPORTS) -w -local github.com/vmware-tanzu ./
+
+$(TOOLS_DIR_RELATIVE)/bin/%: $(TOOLS_DIR_DEPS)
```

Comment:
Could you confirm why you are adding these targets?
Is it to allow to build individual tools more easily? For example, instead of having to do `make tools` we can now just build `counterfeiter` by doing `make hack/tools/bin/counterfeiter`?

That is nice.

Reply:
Yes, it's based on the same stuff I did here https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/Makefile#L407-L409